aws/rust-runtime/aws-config/src/meta/credentials/lazy_caching.rs

```diff
@@ -77,16 +77,17 @@ impl ProvideCredentials for LazyCachingCredentialsProvider {
         future::ProvideCredentials::new(async move {
             // Attempt to get cached credentials, or clear the cache if they're expired
             if let Some(credentials) = cache.yield_or_clear_if_expired(now).await {
-                tracing::debug!("loaded credentials from cache");
+                debug!("loaded credentials from cache");
                 Ok(credentials)
             } else {
                 // If we didn't get credentials from the cache, then we need to try and load.
                 // There may be other threads also loading simultaneously, but this is OK
                 // since the futures are not eagerly executed, and the cache will only run one
                 // of them.
-                let span = trace_span!("lazy_load_credentials");
+                let span = info_span!("lazy_load_credentials");
                 let future = Timeout::new(loader.provide_credentials(), timeout_future);
-                cache
+                let start_time = Instant::now();
```

Comment:
I'm wondering if we should use `tokio::time::Instant` in this so that this is more controllable in tests. Maybe it would be worth testing cache miss time? IDK

Reply:
Using `tokio::time::Instant` in tests is fine, but I'm not as sold on using it in the actual code (without abstracting it in some way) since it requires the `time` feature, which pulls in Tokio's async sleep. I don't think this specific case really needs a test.